src/services/twilio.ts

```diff
@@ -185,6 +185,12 @@ export const isPhoneCodeValid = async ({
   phone: PhoneNumber
   code: PhoneCode
 }) => {
+  if (!isProd) {
+    if (code === "000000") {
```

Comment:
you dont need 2 if to do this validation

Reply:
I'm not sure to understand?